src/test/java/org/mockitousage/basicapi/MockingDetailsTest.java

```diff
@@ -1,54 +0,0 @@
-/*
- * Copyright (c) 2007 Mockito contributors
- * This program is made available under the terms of the MIT License.
- */
-package org.mockitousage.basicapi;
-
-import org.junit.Test;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import org.mockito.Spy;
-import org.mockitoutil.TestBase;
-
-import static junit.framework.TestCase.assertFalse;
-import static junit.framework.TestCase.assertTrue;
-import static org.mockito.Mockito.*;
-
-public class MockingDetailsTest extends TestBase {
```

Comment:
I found this separation confusing, e.g. both of the tests are documentation of usage. It also introduces duplicate coverage and creates unnecessary decision points (where to write tests for mocking details).

Do you want to keep this separation?